x-pack/plugins/actions/server/builtin_action_types/servicenow/config.ts

```diff
@@ -26,13 +26,15 @@ export const snExternalServiceConfig: SNProductsConfig = {
     table: 'incident',
     useImportAPI: ENABLE_NEW_SN_ITSM_CONNECTOR,
     commentFieldKey: 'work_notes',
+    appId: '7148dbc91bf1f450ced060a7234bcb88',
```

Comment:
Does it make sense to create a constants for this ids? Is it any chance this id could be changed?

Reply:
You are right. When the applications are published the ids will stay the same forever. There is a very rare occasion where they can change on publish.